src/execution/executor.ts

```
@@ -0,0 +1,413 @@
+
+  private async executeFileOperation(command: Command): Promise<any> {
+    const operation = command.command;
+    const [target, ...rest] = command.args || [];
+
+    switch (operation) {
+      case 'read':
+        return await fs.readFile(target, 'utf-8');
+      
+      case 'write':
+        const content = rest.join(' ');
+        await fs.writeFile(target, content);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)